source/debian/Demo_User_Guides/Chromium_Browser.rst

```diff
@@ -92,15 +88,15 @@ rendered with this information. The below example shows what is enabled/disabled
     *   WebNN: Disabled
 
 
-If for some reason you suspect the GPU is rending something incorrectly, you can run chromium with GPU disabled
+If, for some reason, you suspect that the GPU is rendering something incorrectly, you can run Chromium with the GPU disabled
 using the :console:`--disable-gpu` flag:
 
 .. code-block:: console
 
     $ chromium https://webglsamples.org/aquarium/aquarium.html --start-fullscreen --disable-gpu
 
 
-To get raw performance numbers from the GPU, you may want to disable frame sync locking in Chromium. This will tell Chromium never to wait for VSYNC and render as fast as the GPU can achieve.
+To get raw performance numbers from the GPU, you may want to disable frame sync locking in Chromium. This will tell Chromium to never wait for VSYNC and render as fast as the GPU can. 
```

Comment:
```suggestion
To get raw performance numbers from the GPU, you may want to disable frame sync locking in Chromium. This will tell Chromium to never wait for VSYNC and render as fast as the GPU can.
```